extensions/brb.md

```diff
@@ -0,0 +1,93 @@
+---
+title: IRCv3 `brb` Extension
+layout: spec
+extends: resume
+copyrights:
+  -
+    name: "Jess"
+    period: "2019"
+    email: "jess@jesopo.uk"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `brb` capability name. Instead, implementations SHOULD
+use the `draft/brb` capability name to be interoperable with other
+software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+Occasionally, clients will need to restart in order to update servers and, at 
+the moment, this will cause a `QUIT` to be sent out for the user on all their 
+networks (unless the client supports in-place upgrading e.g. weechat.)
+
+With the introduction of the `resume` spec, we have an opportunity to leverage 
+the ability to resume an old session to achieve this but we need a way to 
```

Comment:
"to achieve this", may just be because the above paragraph doesn't go too in-depth into the actual solution but it feels like the meaning of 'this' here isn't really defined. Doesn't really clearly explain the issue currently (disruption to the client upgrading and others on the network feels like the main issue?) and how this spec is designed to solve that problem

Reply:
From this: It feels like the 'this' is mostly-silent and un-disruptive software upgrades. If this is what's intended, could we add a sentence to the end of the first paragraph saying something along the lines of... _"This specification introduces a feature that allows clients to do these upgrades in a mostly-silent way, using the `resume` feature."_ Spells out the purpose of the document more cleanly and to-the-point